drivers/display/Kconfig.mcux_dcnano_lcdif

```
@@ -58,4 +58,4 @@ config MCUX_DCNANO_LCDIF_EXTERNAL_FB_ADDR
 
 endif # MCUX_DCNANO_LCDIF_EXTERNAL_FB_MEM
 
-endif # DISPLAY_MCUX_ELCDIF
+endif # DISPLAY_MCUX_LCDIF
```

```suggestion
endif #DISPLAY_MCUX_DCNANO_LCDIF
```

Should match the Kconfig name used on line 13 in this file

---

Thanks, fixed.